SuccessEycaActivationActivity/handler.ts

```diff
@@ -141,8 +141,8 @@ export const getSuccessEycaActivationActivityHandler = (
             .update(_)
             .mapLeft(err => fail(toError(err), "Cannot update EYCA card"))
         )
-        .map(() => success())
+        .map(eycaCard => successWithValue(eycaCard.card))
     )
-    .fold<ActivityResult>(identity, identity)
+    .fold<ActivityResult>(identity, v => v as ActivityResult)
```

Comment:
Why do we need this explicit cast?

Reply:
It doesn't build otherwise. Actually this is a point I'm not 100% sure of: I was expecting to be able to use identity as well

Reply:
You've just to change a little the implementation of `successWithValue`: It seems encoders does not take care about Date typings (transforming dates into string) we can do something like:
```
export const successWithValue = (
  value: SuccessValue
): ActivityResultSuccessWithValue => ({
  kind: "SUCCESS_VAL",
  value
});
```

Reply:
Done in [#9d07ba5](https://github.com/pagopa/io-functions-cgn/pull/27/commits/9d07ba561352276df16d81ca4859d3d33e9c88fa)